### ISSUE TITLE
TNBpay

### ISSUE THREAD

**Issue by olowudaniel1** [thenewboston-developers/Project-Proposals#172]
#### How would you describe your project?
Creating a plugin that would allow users to pay with TNB on popular E-commerce platform such as WooCommerce, Shopify, Wix stores, Bigcommerce and PrestaShop

#### What problem does it solve?
It enables Users to pay with TNB directly for products and services on popular E-commerce platform, hence, eradicating the need to first of all sell their coins in exchange for the required currency.

#### Who is the target market?
TNB users that own E-commerce sites, also, TNB users that participate in online shopping.

#### What do you think is the opportunity of this project?
This project provides the opportunity of having more people involved with TNB once the beta is reached.

#### How does it integrate with thenewboston digital currency work?
This project solely revolves round TNBC and it would bring more people to use TNB.

#### Are there any competitors?
There are other payment plugins that revolves round dollars and BTC but apparently there is none that has to do with TNBC, hence, there are no competitors.

#### Will you need specific information or advice?
Not at the moment

#### What are your goals and objectives?
To have users easily pay for products and services on popular E-commerce platforms with their TNBC
Long term goal: To have a fully implemented SDK for this 
#### How will you evaluate project success?
Number of Stores that have successfully integrated the plugin.

#### What risks should you plan to manage?
There are no risks, the project would be go through series of tests after development to ensure there are no flaws.

#### What are your milestones?
- Milestone 1: Recruit Team Members
-Milestone 2: Architectural design
-Milestone 3: UI/UX Design
-Milestone 4: Backend Development
-Milestone 5: Testing
More Milestones to be determined.

#### Tell us about your team members:
Full Name, Slack Username, GitHub Username, LinkedIn, Role, Team Joining Date, Team Leaving Date, Account number
Olowu Daniel, Daniel Olowu, olowudaniel1, 22-03-2021, -, Project Manager/Team lead, 86ee75d4cea77ca8339cc97f5e7426a3434d9f91465cd441c0c7593bd852e955

#### List the roles of any additional team members you would be looking for?
UI/UX Designer
Frontend Developer
Backend Developer

**Comment by mrbusysky** [comment]
Keysign is currently making this

**Comment by olowudaniel1** [comment]
I just checked keysign, its a browser extension. I am envisioning creating a plugin for popular E-Commerce platforms such as Woo Commerce where Customers with TNB can pay directly for goods and services  just as how they can with Paypal and their credit cards. It won't be a browser extension, It would be a plugin available for each popular E-commerce platform. 

![tnb](https://user-images.githubusercontent.com/79472442/112763065-dc90b880-8ffa-11eb-87aa-4a99183bf489.JPG)

**Comment by mrbusysky** [comment]
Keysign is building a suite of products, extensions, mobile app, desktop app, and website with merchant services like paypal.

**Comment by olowudaniel1** [comment]
Okay. Thanks for the update

**Comment by mrbusysky** [comment]
We do have plugins planned though, so maybe we can work something out. Feel free to message me on discord

**Comment by manishram** [comment]
Hey, @olowudaniel1 if you want to continue this project proposal please update it with the new format.
Check the new project proposal format [HERE](https://github.com/thenewboston-developers/Projects/issues/new?assignees=&labels=Project&template=project-proposal.md&title=NAME_OF_YOUR_PROJECT) or close this proposal in case you want to continue with @mrbusysky

**Comment by olowudaniel1** [comment]
Hi @manishram , thanks for the info. I will update the project ASAP.

**Comment by olowudaniel1** [comment]
Should I edit directly from here or create a new issue?

**Comment by manishram** [comment]
You can edit this one.

**Comment by olowudaniel1** [comment]
@manishram it has been edited

**Comment by manishram** [comment]
Hey @olowudaniel1,
- Need to work on milestones. [We'll provide you better insight on acceptable milestones asap]

**Comment by olowudaniel1** [comment]
Please do, I will update it ASAP

**Comment by olowudaniel1** [comment]
Still waiting for your feedback

**Comment by manishram** [comment]
Hey @olowudaniel1 we'll let you know.

**Comment by angle943** [comment]
Hey @olowudaniel1 , the government has reviewed this project, and we like the idea, but a few things we want to discuss:

1) Let's keep milestone 1 to be just for the wireframe designs and sitemaps. Hi-fi designs can be later. Here are examples of such we are looking for:

Sitemap:
[Sitemap_Wireframe.pdf](https://github.com/thenewboston-developers/Projects/files/6527470/Sitemap_Wireframe.pdf)

Wireframe:
[Landing_Page_Wireframe.pdf](https://github.com/thenewboston-developers/Projects/files/6527471/Landing_Page_Wireframe.pdf)


Once the wireframes (the first milestone) is done, we also recommend that you separate the different platforms (woo commerce, shopify, etc) into separate milestones. Milestones should be easily quantifiable and judged by the auditors

**Comment by olowudaniel1** [comment]
Here is the updated milestone details

#### Milestone Description

1. User flow and wireframing
2. Wordpress development
3. Wix store development
4. Shopify development
5. Bigcommerce development
6. PretaShop development

**Comment by angle943** [comment]
hello @olowudaniel1 , we have reviewed your project, and we have approved milestone 1 (User flow and wireframing) for 5,000 coins!
